main.go

```diff
@@ -32,7 +33,7 @@ import (
 )
 
 func main() {
-	fmt.Printf("\nCrossdock starting...\n\n")
+	fmt.Printf("\nCrossdock starting (go %v)...\n\n", runtime.Version())
```

Comment:
As you wish. It was tricky to debug what version of go was running in the container, and I am not sure anybody would care about the go version in the output until you have to troubleshoot.